src/options.js

```diff
@@ -5,7 +5,20 @@ import {color, descending, quantile} from "d3";
 const TypedArray = Object.getPrototypeOf(Uint8Array);
 const objectToString = Object.prototype.toString;
 
-// This allows transforms to behave equivalently to channels.
+/**
+ * Given an iterable *data* and some *value* accessor, returns an array (a column) of the specified *arrayType* with the corresponding value of each element of the data. The *value* accessor may be one of the following types:
```

Comment:
Thoughts on hard wrapping? I like to not use hard wrapping Markdown, but I like hard wrapping TypeScript/JavaScript. It’s not a big deal to have the hard wrapping in Markdown if it’s going to be auto-generated, though (and we could also have the script remove hard wrapping, perhaps).

Reply:
We've discussed it earlier today with Duane, and arrived at that very conclusion. Unformatted descriptions looks too weird in ts files, and since the copy/paste/format operation is going to be automated it's the best choice.

Reply:
Yep! My plan is to wrap at 120 characters (could do 100 or 80 if preferred). I didn't plan to remove those hard wraps when injecting into the README though, but that would be doable if we really wanted that (though I would need to double-check the markdown rules around hard-wrapping).

Another thing I considered was to do some pseudo-two-way-binding between the README and the JSDoc:detect (via git) whether the changes changes made in the branch were to jsdoc sections of the README and if so reverse the direction and update the JSDoc from the README. Decided that's a bit much though

Reply:
Is this all that remains to do?

Reply:
This is done in #1034